content/ar-sandbox.md

```diff
@@ -35,47 +31,41 @@ For detailed information on the hardware setup, see the [AR Sandbox hardware tut
 
 - Roughly 200 pounds of white sand like [Sandtastik White Sandbox Sand](https://www.amazon.com/Sandtastik-White-Play-Sand-SND025/dp/B001AZ0CGG)
 
-- A Linux-friendly computer with a fast NVIDIA GPU, running Pop!_OS/Ubuntu 18.04 or newer (any flavor).
+- A Sysytem76 computer with a fast NVIDIA GPU, running Pop!_OS 22.04 or newer (any flavor).
 
-In terms of System76 computers we recommend the [Oryx Pro](https://system76.com/laptops/oryx) laptop or [Thelio](https://system76.com/cart/configure/thelio-r1) desktop with the fastest available GPU.
+In terms of System76 computers we recommend the [Oryx Pro](https://system76.com/laptops/oryx) laptop or [Thelio](https://system76.com/desktops) desktop with a dedicated GPU.
```

Comment:
Is any GPU okay or will it require NVIDIA?

Reply:
I _believe_ Brock actually got it working reasonably well on his Dev One yesterday. If integrated graphics is powerful enough to run this nowadays, then that might simplify this project quite a bit.

Reply:
Yes I did get it working on my Dev one. I would still recommend a dGPU though, performance was much better in the oryp6